theano/gpuarray/c_code/dimshuffle.c

```diff
@@ -0,0 +1 @@
+#section support_code
```

Comment:
Not sure that alloca is portable across systems and compilers:
- http://man7.org/linux/man-pages/man3/alloca.3.html#NOTES
- https://github.com/Theano/Theano/issues/6094